dashboards-observability/public/components/application_analytics/home.tsx

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+import React, { useEffect, useState } from 'react';
+import { AppTable } from './components/app_table';
+import { Application } from './components/application';
+import { CreateApp } from './components/create'
+import { Route, RouteComponentProps, Switch } from 'react-router';
+import { TraceAnalyticsComponentDeps, TraceAnalyticsCoreDeps } from '../trace_analytics/home';
+import { FilterType } from '../trace_analytics/components/common/filters/filters';
+import DSLService from 'public/services/requests/dsl';
+import PPLService from 'public/services/requests/ppl';
+import SavedObjects from 'public/services/saved_objects/event_analytics/saved_objects';
+import TimestampUtils from 'public/services/timestamp/timestamp';
+import { handleIndicesExistRequest } from '../trace_analytics/requests/request_handler';
+import { ObservabilitySideBar } from '../common/side_nav';
+import { NotificationsStart } from '../../../../../src/core/public';
+
+export interface AppAnalyticsCoreDeps extends TraceAnalyticsCoreDeps {}
+
+interface HomeProps extends RouteComponentProps, AppAnalyticsCoreDeps {
+  pplService: PPLService;
+  dslService: DSLService;
+  savedObjects: SavedObjects;
+  timestampUtils: TimestampUtils;
+  notifications: NotificationsStart;
+}
+
+export interface AppAnalyticsComponentDeps extends TraceAnalyticsComponentDeps {}
+
+export type ApplicationType = {
+  name: string;
+  id: string;
+  composition: string;
+  currentAvailability: string;
+  availabilityMetrics: string;
+  dateCreated: string;
+  dateModified: string;
+};
+
+const dateString = new Date().toISOString();
+
+const dummyApplication: ApplicationType[] = [{
+  name: "Cool Application", 
+  id: "id", 
+  composition: "Payment, user_db",
+  currentAvailability: "Available",
+  availabilityMetrics: "Error rate: 0.80%, Throughput: 0.94%, Latency: 600ms",
```

Comment:
Are we pushing dummy data to main branch? If App Analytics is fully functional we can remove this.

Reply:
if this has to be here, maybe put it in a separate mock data directory and then import the constant?

Reply:
App Analytics is not fully functional yet so we have dummy data for now.

Reply:
@joshuali925 If the feature is still in development should we have a separate branch for it?

Reply:
following opensearch standards we can merge to main, but i think it's better to keep in a separate branch during development so any external user won't get a broken UI using default branch

Reply:
Just create branch application-analytics and change to merge into there!